package.json

```diff
@@ -7,7 +7,8 @@
     "lint": "eslint \"src/**/*.js\" \"test/**/*.js\"",
     "test": "npm run lint && npm run build",
     "build": "webpack -p",
-    "start": "webpack-dev-server -d --inline --hot"
+    "start": "webpack-dev-server -d --inline --hot",
+    "prepublish": "babel -d dist/ src/"
```

Comment:
Thinking more about it: we don't follow NPM / Node.js rules for imports (non-relative names –i.e. those not starting with `./` or `../`– are resolved against `node_modules` sibling folders walking up the hierarchy; so `require("Core/Commander/Command")` wouldn't work in "pure Node.js", it'd have to be either `require("itowns2/lib/Core/Command/Command")` to resolve to the file from an _installed_ `itowns2` NPM module; or `require("./Command")` –or `import Command from './Command'`– when called from our `InterfaceCommander`). See http://www.commonjs.org/specs/modules/1.0/ and https://nodejs.org/api/modules.html for the details.

Our current usage of referencing files by their _absolute_ path “relative” to our `src` root only works because we configured Webpack that way, but it won't/wouldn't work if we publish our files to NPM and try to use the module from another project (if using Webpack or Rollup –and probably also Browserify–, one could configure it to make it work, but that's additional burden).

So I think we should just remove that `prepublish` script, and revisit things when we want to tackle the NPM issue (see #44).  
(FWIW, I also think we should move to using relative import paths, like everyone else in the JS/Node.js land, but again, that's another matter that should be addressed in a distinct PR)